datastore/zmq/zmq.go

```diff
@@ -0,0 +1,42 @@
+package zmq
+
+// DefaultTopicSep is the default join character used to build a topic.
+const DefaultTopicSep = "."
+
+// Config contains the data necessary to configure a zmq producer.
+type Config struct {
```

Comment:
can you add a unit test for config similar to https://github.com/teslamotors/fleet-telemetry/blob/main/config/config_test.go#L117?